File: widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/ShowScreenSharingNotificationUseCase.java
package com.glia.widgets.core.notification.domain;

import com.glia.widgets.core.notification.device.INotificationManager;

public class ShowScreenSharingNotificationUseCase {
    private final INotificationManager notificationManager;

    public ShowScreenSharingNotificationUseCase(INotificationManager notificationManager) {
        this.notificationManager = notificationManager;
    }

    public void execute() {
        this.notificationManager.showScreenSharingNotification();
    }
}

File: widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/ShowScreenSharingNotificationUseCase.kt
package com.glia.widgets.core.notification.domain

import com.glia.widgets.core.notification.device.INotificationManager

internal class ShowScreenSharingNotificationUseCase(private val notificationManager: INotificationManager) {
    operator fun invoke() = notificationManager.showScreenSharingNotification()
}
